lib/utils/note-utils.js

```diff
@@ -7,8 +7,19 @@
  *
  * @type {RegExp} matches a title and excerpt in note content
  */
+//empty group at the beginning is intended to match group indexes with regex for Markdown notes
```

Comment:
let's move this into the docblock already present.

```js
/**
 * Matches the title…
 *
 * …
 *
 * Note! there is an "extra" capture group of whitespace at the
 * beginning of the pattern. This is there so that we can maintain
 * parity among the capture group indices with the pattern for
 * notes where Markdown is active.
 *
 * @type {RegExp} …
 */
const noteTitleAndPreviewRegExp = …
```

Reply:
this isn't valid since my previous commit. I've removed this note.